### ISSUE TITLE
Cells/background not rendered

### ISSUE THREAD

**Issue by RougeExciter** [kolinkrewinkel/KKGridView#141]
I must be doing something stupid wrong!

XCode 4.3.2, iPhone, 5.1 or above, Story board with KKGridViewController sub-classed. 

All the section headers are drawing correctly (and are sized correctly for the content) and although the pressed and selected states are correctly drawn (cells turn blue), every other cell is white regardless of what I do. I've checked background color observers are firing etc, but other wise just big white sections with cells that turn blue only when tapped. cell.imageView.image has no effect. 

Master branch, downloaded on Thursday. 

What am I doing dumb?

**Comment by kolinkrewinkel** [comment]
Hmm...I've noticed a lot of people having issues with Nib-based grid view instances.  Can you try to implement it in code and let me know the result?

**Comment by RougeExciter** [comment]
You are right, it's working fine when implemented directly. I'll try and get the time to create a smaller storyboard example that I can send rather than the rather large product project.
